### ISSUE TITLE
python-pillow v10.0.0 breaks pysol

### ISSUE THREAD

**Issue by mlmatlock** [shlomif/PySolFC#323]
Arch system, fully UTD.  Upgrading to pillow v10.0.0 produces this:

```
pysol
Traceback (most recent call last):
  File "/usr/bin/pysol", line 36, in <module>
    from pysollib.main import main  # noqa: E402,I202
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/pysollib/main.py", line 30, in <module>
    from pysollib.app import Application
  File "/usr/lib/python3.11/site-packages/pysollib/app.py", line 32, in <module>
    from pysollib.actions import PysolMenubar
  File "/usr/lib/python3.11/site-packages/pysollib/actions.py", line 28, in <module>
    from pysollib.help import help_about, help_html
  File "/usr/lib/python3.11/site-packages/pysollib/help.py", line 29, in <module>
    from pysollib.pysoltk import HTMLViewer
  File "/usr/lib/python3.11/site-packages/pysollib/pysoltk.py", line 28, in <module>
    from pysollib.ui.tktile.tkutil import *  # noqa: F401,F403
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/pysollib/ui/tktile/tkutil.py", line 250, in <module>
    class PIL_Image(ImageTk.PhotoImage):
  File "/usr/lib/python3.11/site-packages/pysollib/ui/tktile/tkutil.py", line 291, in PIL_Image
    def resize(self, xf, yf, resample=Image.ANTIALIAS):
                                      ^^^^^^^^^^^^^^^
AttributeError: module 'PIL.Image' has no attribute 'ANTIALIAS'
```
Returning pillow to v9.5 fixes.

**Comment by joeraz** [comment]
This was resolved in commit 75a3000ca1301e15d5fdbc6bae6df8dbf1cac066, but has not yet been released due to an issue with the continuous integration processes getting an error installing Pillow 10.0.0, which prevents generating packages.

**Comment by joeraz** [comment]
The fix was released in version 2.21.0.
